src/bin/cargo/commands/owner.rs

```diff
@@ -6,25 +6,68 @@ use cargo::util::auth::Secret;
 pub fn cli() -> Command {
     subcommand("owner")
         .about("Manage the owners of a crate on the registry")
-        .arg_quiet()
         .arg(Arg::new("crate").action(ArgAction::Set))
-        .arg(
-            multi_opt(
-                "add",
-                "LOGIN",
-                "Name of a user or team to invite as an owner",
-            )
-            .short('a'),
-        )
-        .arg(
-            multi_opt(
-                "remove",
-                "LOGIN",
-                "Name of a user or team to remove as an owner",
-            )
-            .short('r'),
+        .arg_required_else_help(true)
```

Comment:
Technically, this is a breaking change though it should be fine.  Please call this breaking change in the PR description though

Reply:
I already indicated in the description that this could be a big change.

Reply:
All I'm noticing is a bolded, many "!" "this may be a big change".  Instead I would expect a section to specifically call out that there is a breaking change, what it is, and why we feel its safe to do.

Reply:
I revised the description, hoping it was right this time.

Reply:
Maybe I missed it but I didn't see what I was asking for.  I've edit the PR description so you have an example of what we are expecting